core/src/main/java/org/stellar/anchor/sep6/Sep6Service.java

```diff
@@ -357,8 +360,10 @@ public StartWithdrawResponse withdrawExchange(
     }
 
     StellarAssetInfo sellAsset = requestValidator.getWithdrawAsset(request.getSourceAsset());
+    String fundingMethod =
+        request.getFundingMethod() != null ? request.getFundingMethod() : request.getType();
     requestValidator.validateTypes(
-        request.getType(), sellAsset.getCode(), sellAsset.getSep6().getWithdraw().getMethods());
+        fundingMethod, sellAsset.getCode(), sellAsset.getSep6().getWithdraw().getMethods());
```

Comment:
Do we not need to persist this field in the database?

Reply:
In db it's stilled saved as `type`. I'm not gonna rename it for now (at least in this pr) as type is a preserved word, and it's causing a lot of issue when I was trying.

Reply:
If only `funding_method` is specified in the request, will the `type` field in the database get updated? It doesn't look like `Sep6TransactionBuilder `  call is updated.

Reply:
```
    String fundingMethod =
        request.getFundingMethod() != null ? request.getFundingMethod() : request.getType();
    requestValidator.validateTypes(
        fundingMethod, asset.getCode(), asset.getSep6().getDeposit().getMethods());
```
and we pass this calculated `fundingMethod` to `Sep6TransactionBuilder.type()`

Reply:
It looks `type` is still set to `request.getType()` on line 43. Can you point to me where it's passed into type, maybe I'm missing something?

Reply:
My bad.

Reply:
It would be great if we can include a unit test that tests that `funding_method` and `type` are persisted to the DB.

Reply:
Also, `type` is present in the platform transaction response. Should we rename that to `funding_method`?

Reply:
The platform transaction response is closely tied to platformTransactionData, and I prefer to avoid changing the name of the data column